tests/func/test_run_single_stage.py

```diff
@@ -898,6 +898,8 @@ def setUp(self):
         super().setUp()
         ret = main(["config", "cache.type", "hardlink"])
         self.assertEqual(ret, 0)
+        self.dvc.close()
+
         self.dvc = DvcRepo(".")
```

Comment:
I did not want to waste too much time on why `self.dvc.config.load()` was not working. So I am going for close and reopen here.